.github/workflows/continous-integration.yml

```diff
@@ -162,15 +162,33 @@ jobs:
         user: __token__
         password: ${{ secrets.PYPI_TOKEN }}
 
-    - name: Publish Release Notes 🗞
-      if: env.IS_TAG_BUILD
+    - name: Notify Slack & Publish Release Notes 🗞
+      uses: Ilshidur/action-slack@2.1.0
+
       env:
+        RASA_SDK_VERSION: ${{ github.ref }}
+        SLACK_WEBHOOK: ${{ SLACK_WEBHOOK }}
         GITHUB_TAG: ${{ github.ref }}
         GITHUB_REPO_SLUG: ${{ github.repository }}
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       run: |
```

Comment:
You can't use `uses` together with `run`. In this case, where you use a GH Action, we don't need to use the `run` argument.

Reply:
Thank you. 👍🏻 That's the reason why the CI workflow didn't even start.